src/app/code-set/code-set-details/code-set-details.component.html

```diff
@@ -283,19 +274,19 @@
 
 <div
   *ngIf="
-    showSecuritySection && !editableForm.visible && !editableForm.deletePending
+    showSecuritySection && !editMode && !deleteInProgress
```

Comment:
Security sections `<div>` elements shouldn't be needed anymore now that the user access/permissions opens a modal dialog.